modules/kubernetes-addons/adot-collector-haproxy/main.tf

```diff
@@ -9,7 +9,13 @@ module "helm_addon" {
   depends_on = [kubernetes_namespace_v1.collector]
 }
 
+moved {
```

Comment:
I don't know if we're ready to use move blocks but if we do, we'll probably add them under a `migrations.tf` file

Reply:
Happy to remove them, if you'd prefer? I have found them pretty helpful. I was just worried that renaming a namespace would be quite destructive, for not much gain.

Reply:
Removed the move blocks for now.